### PR TITLE
Add wording to line 38

### PR DESCRIPTION
**Pull request by joroar**
The first section of the article says this:
"By default, script is allowed on sites that admins create. It is not allowed on OneDrive, on sites users create themselves, and on the root site for your organization."

That is true, but by default, it is also not allowed on Communication and Modern Team sites.  The DenyAddAndCustomizePages property is set to Enabled on those sites by default.  This omission has led to customers opening support cases.

I believe the wording should be updated to something like this:
"By default, script is allowed on most sites that admins create. It is not allowed on OneDrive, on sites users create themselves, on modern Team and Communications sites, and on the root site for your organization."

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/allow-or-prevent-custom-script.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ As a global or SharePoint admin in Microsoft 365, you can allow custom script as
 > [!NOTE]
 > For simple ways to change the look and feel of a site, see [Change the look of your SharePoint site](https://support.office.com/article/06bbadc3-6b04-4a60-9d14-894f6a170818). 
   
-By default, script is allowed on sites that admins create. It is not allowed on OneDrive, on sites users create themselves, and on the root site for your organization. You'll probably want to limit the amount of script you allow for security reasons. For more info about the security implications of custom script, see [Security considerations of allowing custom script](security-considerations-of-allowing-custom-script.md).
+By default, script is allowed on most sites that admins create. It is not allowed on OneDrive, on sites users create themselves, on modern Team and Communications sites, and on the root site for your organization. You'll probably want to limit the amount of script you allow for security reasons. For more info about the security implications of custom script, see [Security considerations of allowing custom script](security-considerations-of-allowing-custom-script.md).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/allow-or-prevent-custom-script.md:38]:
```suggestion
By default, script is allowed on most sites that admins create. It is not allowed on OneDrive, on sites users create themselves, on modern team and communications sites, and on the root site for your organization. You'll probably want to limit the amount of script you allow for security reasons. For more info about the security implications of custom script, see [Security considerations of allowing custom script](security-considerations-of-allowing-custom-script.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**